tools/device_file_generator/stm_writer.py

```diff
@@ -194,6 +194,7 @@ def addGpioToNode(self, node):
 					gpio_child = driver.addChild('gpio')
 					gpio_child.setAttributes(attr)
 					gpio_child.setAttributes(gpio)
+					
```

Comment:
also, in this case this would be bad style in any case, as this changes a file that is not involved in your actual changes and thus contributes unnecessary noise to your commit